src/locales/en.json

```
@@ -28,6 +28,7 @@
     }
   },
   "header": {
+    "home-link": "Openverse home",
```

```suggestion
    "home-link": "Home",
```

I think just "home" is fine.